main.js

```diff
@@ -14,48 +14,43 @@ class Libro {
     this.title = title;
     this.author = author;
   }
-}
 
-function deleteBook(index, libro) {
-  libro.remove();
-  dataBooks.splice(index, 1);
-  localStorage.setItem('book', JSON.stringify(dataBooks));
+  deleteBook(index) {
+    this.dataBooks.splice(index, 1);
+    localStorage.setItem('book', JSON.stringify(dataBooks));
+  }
+
+  addBook(libro) {
+    this.dataBooks.push(libro);
+    localStorage.setItem('book', JSON.stringify(dataBooks));
+  }
 }
 
 const printList = () => {
+  const libro = new Libro();
   books.innerHTML = null;
   if (localStorage.getItem('book')) {
     dataBooks = JSON.parse(localStorage.getItem('book'));
   }
   for (let i = 0; i < dataBooks.length; i += 1) {
     const bookStorage = document.createElement('div');
-    const bookName = document.createElement('p');
-    const authorName = document.createElement('p');
+    const bookInfo = document.createElement('p');
     const deleteBtn = document.createElement('button');
-    bookName.textContent = dataBooks[i].title;
-    authorName.textContent = dataBooks[i].author;
-    deleteBtn.textContent = 'Delete';
+    bookInfo.textContent = `"${dataBooks[i].title}" by ${dataBooks[i].author}`;
+    deleteBtn.textContent = 'Remove';
     books.appendChild(bookStorage);
-    bookStorage.appendChild(bookName);
-    bookStorage.appendChild(authorName);
-    bookStorage.appendChild(deleteBtn);
+    bookStorage.append(bookInfo, deleteBtn);
+    bookStorage.className = 'bookStorage';
     deleteBtn.addEventListener('click', () => {
-      deleteBook(i, bookStorage);
+      bookStorage.remove();
+      libro.deleteBook(i);
     });
   }
 };
 
 printList();
 
-const addBook = (title, author) => {
-  const libro = new Libro(title, author);
-  dataBooks.push(libro);
-  localStorage.setItem('book', JSON.stringify(dataBooks));
-  printList();
-};
-
 addBtn.addEventListener('click', () => {
-  addBook(titulo.value, autor.value);
-  titulo = '';
-  autor = '';
-});
+  const libro = new Libro(titulo.value, autor.value);
+  libro.addBook(libro);
+});
```

Comment:
- [ ] The new book is not added to the local storage and is not displayed on the page after clicking the add button. Please work on it.

Reply:
Hi @medaminedev66, 
Can you please help me to solve some linters issues that our code has?
Regards.

Reply:
Hi @AntonioHincapie. Yes, of course. What is the problem with your code? Could you be much more specific?

Reply:
> Hi @AntonioHincapie. Yes, of course. What is the problem with your code? Could you be more specific?

![image](https://user-images.githubusercontent.com/99298594/165435009-5cba71d9-42d7-48da-a9b8-2a9739ae9bf9.png)

Hi @medaminedev66 
Look this is the problem that shows the linters on the main.js file on the last commit.
Please can you help me to solve this?
Regards.

Reply:
Ok let me check your code

Reply:
Hi @AntonioHincapie, you have deleted the variable declaration `dataBooks` from your code, but you're still using it in the class methods of Liblo class and in the first condition of your code, despite the fact that you can't use an undefined variable in JavaScript.
As I mentioned in the previous review comment, although you cannot define arrays or objects in your code, you may define them in classes.